.github/workflows/CI.yml

```diff
@@ -14,9 +14,12 @@ jobs:
           apt update
           apt install -y sudo
           sudo apt install -y git
-          sudo DEBIAN_FRONTEND=noninteractive apt install --yes npm
-      - name: Install commitlint
-        run: npm install
+
+          sudo apt install -y curl
+          # can't install ubuntu's default nodejs version because we would get this error:
+          # error @commitlint/config-conventional@17.1.0: The engine "node" is incompatible with this module. Expected version ">=v14". Got "12.22.9"
```

Comment:
@realmarv this is not the same error explained in the commit msg

Reply:
It seems there is no error in the sanitycheck job

Reply:
Right fixed it